docs/pages/reference/mesheryctl-commands.md

```diff
@@ -129,6 +129,19 @@ Installation, troubleshooting and debugging of Meshery and its adapters.
         <td>{{ flag.description }}</td>
       </tr>
     {% endfor %}  
+  {% endfor %}
```

Comment:
@umairnsr87 Could you please delete this one too? As it is not relevant to the issue